snap/snapcraft.yaml

```diff
@@ -76,6 +76,7 @@ apps:
     daemon: simple
     install-mode: disable
     refresh-mode: endure
```

Comment:
Now that we handle stop/refresh carefully we can remove `refresh-mode` 'endure'.